lms/envs/devstack.py

```diff
@@ -20,7 +20,8 @@
 HTTPS = 'off'
 
 LMS_ROOT_URL = 'http://localhost:8000'
-ENTERPRISE_API_URL = LMS_ROOT_URL + '/enterprise/api/v1/'
+LMS_INTERNAL_ROOT_URL = LMS_ROOT_URL
```

Comment:
@mtyaka After testing this on the docker devstack, I think we'll need this in `lms/envs/devstack_docker.py`, too, but for port 18000.